### PR TITLE
Added Partial clone

### PR DESCRIPTION
**Pull request by tknippenberg**

### CODE REVIEW

**Comment by tknippenberg** [comment]
- Future release (aiming for 10.12)
- Technical review needed (@blis-mendix)

**Review by blis-mendix — COMMENTED** [review]
Technically looks good

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/clone-type.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Clone type"
+url: /refguide/clone-type/
+weight: 20
+#If moving or renaming this doc file, implement a temporary redirect and let the respective team know they should update the URL in the product. See Mapping to Products for more details.
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/clone-type.md]: @tknippenberg , this note means that the document is cross-referenced from the product. Did the team add a link to this doc in SP?

Comment (tknippenberg) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/clone-type.md]: There is no direct link from the product. I will need to link to it from external pages (like the blog etc). Will that be possible?

Comment (tknippenberg) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/clone-type.md]: Ah you specifically mean the comment on line 5? That can indeed be removed. I think I just copy-pasted it from another page.

File: content/en/docs/refguide/version-control/using-version-control-in-studio-pro/clone-type.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+There are several advantages to this clone type:
+* A full copy allows you to do some operations on the command line without contacting the server. As Git operations like a `detached head` are not supported in Studio Pro, the benefits are limited. 
+* Operations like [combining changes](/refguide/merge-algorithm/) can be significantly faster when working with a full clone. Even though the latest changes still have to be downloaded from the server, your local repository already contains a part of it, which saves time.
+
+The disadvantage of this close type is that downloading a full clone can take significantly longer. A 2GB repository needs to be completely downloaded and unpacked, even though the usage of the full history locally is very limited.
+### 2.2 Partial Clone
+
+{{% alert color="info" %}}
+Partial clone is beta in Mendix version 10.12 and above. For more information on beta, see [Beta and Experimental Releases](/releasenotes/beta-features/).
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/version-control/using-version-control-in-studio-pro/clone-type.md:34]: @tknippenberg , I added a beta note here. Or should it be added for the whole doc at the very top?

Comment (tknippenberg) [reply content/en/docs/refguide/version-control/using-version-control-in-studio-pro/clone-type.md:34]: I think it's sensible in this location 👍

**Review by MariaShaposhnikova — APPROVED** [review]

**Pull request merged**